hbase-client/src/main/java/org/apache/hadoop/hbase/RegionMetrics.java

```diff
@@ -160,4 +160,9 @@ default String getNameAsString() {
    *   of this region
    */
   int getMaxCompactedStoreFileRefCount();
+
+  /**
+   * @return the data locality for ssd of region in the regionserver.
```

Comment:
It's not all that clear what the "data locality for ssd" means without looking at the code.

Reply:
Thanks for review,will fix it.

Reply:
What about add a desc for method like this "Different from dataLocality,this metric only include the data stored on ssd"?

Reply:
or "Different from dataLocality,the numerator only include the data stored on ssd"?